<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(180, 255, 229)</string>
  </property>
  <widget class="QLineEdit" name="m1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mt1">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mt2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mt3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mt4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(138, 177, 255)</string>
   </property>
   <property name="text">
    <string>Medication</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(138, 177, 255)</string>
   </property>
   <property name="text">
    <string>Time to Dispense</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshMedbutton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(41, 92, 223)</string>
   </property>
   <property name="text">
    <string>Update Information</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backtoMain3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:red</string>
   </property>
   <property name="text">
    <string>Back to Main</string>
   </property>
  </widget>
  <widget class="QPushButton" name="verifyItsYou">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>300</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:red</string>
   </property>
   <property name="text">
    <string>Verify It's You</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
